05/app.js

```diff
@@ -8,8 +8,21 @@ const stats = {
 };
 
 /* tutaj umieść swój kod */
+const pElement = document.querySelectorAll("p");
+pElement.forEach((e) => {
+	const links = e.children;
+	const linksArr = Array.from(links);
 
-
+	e.addEventListener("click", function () {
+		stats.paragraphs.p1++;
+	});
+	linksArr.forEach((element) => {
+		element.addEventListener("click", function (event) {
+			event.preventDefault();
+			stats.links["/dolor.html"]++;
+		});
+	});
```

Comment:
To działa tylko dla p1 (czyli pierwszy paragraf), a ma działać dla każdego i dla każdego zliczać osobno statystyki.
Postaraj się poprawić i zwróć uwagę, że najpierw trzeba utworzyć taki element dynamicznie (jeśli nie istnieje), a potem do niego dodawać.